GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -112,7 +112,7 @@ private LostObject(LostObjectType objectType, string rawType, ObjectId objectId)
                     {
                         result.Author = module.ReEncodeStringFromLossless(logPatternMatch.Groups[1].Value);
                         string encodingName = logPatternMatch.Groups[2].Value;
-                        result.Subject = module.ReEncodeCommitMessage(logPatternMatch.Groups[3].Value, encodingName);
+                        result.Subject = module.ReEncodeCommitMessage(logPatternMatch.Groups[3].Value, encodingName) ?? "";
```

Comment:
`ReEncodeCommitMessage` mustn't return `null`.

Reply:
The method was previously not annotated `string?` but returned `ReEncodeStringFromLossless()` that returns `string?`. Some calls assumed `string?`, other not.

Do you want to change the implementation in `ReEncodeCommitMessage()` to return `?? ""`?

Reply:
I'd look at changing the implementation of `ReEncodeCommitMessage` to never return `null` if inputs aren't `null`. May be we should add an attribute [`NotNullIfNotNull`](https://docs.microsoft.com/dotnet/csharp/language-reference/attributes/nullable-analysis#specify-conditional-post-conditions-notnullwhen-maybenullwhen-and-notnullifnotnull)?